client/app/styles/base.styl

```diff
@@ -0,0 +1,91 @@
+@font-face
+    font-family: 'SourceSansPro'
+    src: url('/fonts/source-sans-pro.ttf')
+    color: #333
+
+html
+body
+    height: 100%
+    margin: 0
+    padding: 0
+
+
+html
+    font-family: 'SourceSansPro'
+    color: #32363f
+
+
+ul
+    list-style: none
+    padding: 0
+    margin: 0
+
+
+[role="application"]
+    display: flex
+    flex-direction: row
+    height: 100%
+    font-size: 1em
+
+    background-color: rgba(78, 91, 105, 0.25)
+
+[role="navigation"]
+    display: flex
+    flex-direction: column
+
+    min-width: 14em
+    background-color: #f5f6f7
+    box-shadow: inset -1px 0 0 0 rgba(50, 54, 63, 0.16)
+
+[role="navigation"] li
+    display: flex
+    flex-direction: row
+
+[role="navigation"] nav
+    flex: 1
+
+[role="application"] main
+    flex: 1
+    padding: 2.5em 3em
+
+
+
+[role="application"] h1
+    font-size: 1.5em
```

Comment:
Tip: as styles files are preprocessed by _Stylus_, don't hesitate to nest selectors, it'll be more readable / maintainable